graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/analysis/HasPath.kt

```diff
@@ -0,0 +1,47 @@
+package com.anaplan.engineering.azuki.graphs.analysis
+
+import com.anaplan.engineering.azuki.core.runner.AnalysisScenario
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+class HasPath : GraphScenario() {
+
+    @AnalysisScenario
+    fun noPath() {
+        given {
+            thereIsAnUndirectedGraph(graphA) {
+                vertex('a')
+                vertex('b')
+            }
+        }
+        then {
+            hasPath(graphA, 'a', 'b', false)
```

Comment:
🚫 I'd rather have two separate functions, e.g. `hasPath` and `doesNotHavePath`, rather than a boolean parameter.